docs/dev/debugging.rst

```diff
@@ -0,0 +1,40 @@
+Debugging
+=========
+
+
+.. _kerneloops:
```

Comment:
We usually add a prefixed based on the page path to make ref labels unique, for example `_dev-debugging-kernel-oops`

Reply:
I've added the prefix as you suggested and changed the link in the getting started.